src/Compilers/Core/Portable/Generated/Operations.xml.Generated.cs

```diff
@@ -2004,6 +2004,205 @@ public override void Accept(OperationVisitor visitor)
     }
 
     /// <remarks>
+    /// Represents an interpolated string expression.
+    /// </remarks>
+    internal abstract partial class BaseInterpolatedStringExpression : Operation, IInterpolatedStringExpression
```

Comment:
generator is not in yet. working on something else. I will put generator later, but named it as generated since I didn't want people to put random one in this file.